port/win32/omrsl.c

```diff
@@ -178,8 +180,8 @@ omrsl_open_shared_library(struct OMRPortLibrary *portLibrary, char *name, uintpt
 		return 0;
 	}
 
-	/* Now try with "altered search path" to catch DLLs with dependencies */
-	dllHandle = LoadLibraryExW(unicodeName, NULL, LOAD_WITH_ALTERED_SEARCH_PATH);
+	/* LoadLibrary will try appending .DLL if necessary */
```

Comment:
for consistency with the previous comment
```suggestion
	/* LoadLibrary will try appending .DLL if necessary. */
```